src/Sets/Zonotope.jl

```diff
@@ -406,3 +406,36 @@ function _split(Z::Zonotope, gens::AbstractVector, n::AbstractVector)
     end
     return Zs
 end
+
+@inline function _angles(point::AbstractVector{N})  where{N<:Real}
+    (atan(point[2], point[1]) /π * 180 + 360)%360
+end
+
+function _single_quadrant_vertices_enum(G::AbstractMatrix{N}, sorted::Bool=true) where{N<:Real}
+    if !sorted
+        G = sortslices(G, dims=2, by=_angles)
+     end
+    return VPolygon(2*cumsum(hcat(G, -G), dims=2) .- sum(G, dims=2))
```

Comment:
this one line is used to replace https://github.com/JuliaReach/LazySets.jl/blob/master/src/Interfaces/AbstractZonotope.jl#L433-L438 
from my testing, when the number of generators on the scale of 100, this implementation would be much faster. as the size of index in previous implementation growths in the scale of `p^2`

Reply:
```julia
2*cumsum(hcat(G, -G), dims=2) .- sum(G, dims=2)
```
is the same as 
```julia
           p = size(G, 2)
            index = ones(p, 2*p)
            @inbounds for i in 1:p
                index[i, i+1:i+p-1] .= -1
            end
            index[:, 1] .= -1
            G * index
```